packages/react/src/components/Modal/Modal.stories.tsx

```diff
@@ -1,10 +1,12 @@
 import type { Meta, StoryFn } from '@storybook/react';
 import { useRef, useState } from 'react';
 
-import { Button, Combobox, Divider, Paragraph, Textfield } from '..';
+import { Button, Combobox, Divider, Heading, Paragraph, Textfield } from '..';
 
 import { Modal } from '.';
 
+const Footer = (props: JSX.IntrinsicElements['div']) => <div {...props} />;
```

Comment:
Any reason we create this component and use it in one example?
Can we remove this? 
```suggestion
```

Reply:
Good catch, was tmp :o